system/t04_mirror/EditMirror10Test_gold

```diff
@@ -0,0 +1,2 @@
+Downloading ftp://ftp.ru.debian.org/debian/dists/wheezy/Release...
```

Comment:
I think it should be downloading here from `ftp.ch.debian.org` here? (New URL?)

Reply:
Indeed that's not correct. Preparation of archiveRoot was missing. Should be fixed now.